docs/platform/concepts/maintenance-window.rst

```diff
@@ -24,4 +24,12 @@ In case of **Apache Kafka®** and **OpenSearch®** the service DNS address resol
 Maintenance updates
 ~~~~~~~~~~~~~~~~~~~
 
-Security updates, platform updates that affect reliability or stability of the service nodes, and quarterly patch updates are always mandatory. Other updates are initially optional. Advance notice is given for all updates. After optional updates have been available for six months, they become mandatory and are applied on the next week's maintenance window at the earliest. This means you have at least 7 days advance notice with exception of critical security updates. These critical updates are applied in the maintenance window of the current week. During service upgrades, maintenance updates are automatically applied and do not require any action from you. 
\ No newline at end of file
+Security updates, platform updates that affect reliability or stability of the service nodes, and quarterly patch updates are always mandatory. Other updates are initially optional. Advance notice is given for all updates. After optional updates have been available for six months, they become mandatory and are applied on the next week's maintenance window at the earliest. This means you have at least 7 days advance notice with exception of critical security updates. These critical updates are applied in the maintenance window of the current week. During service upgrades, maintenance updates are automatically applied and do not require any action from you.
+
+Pending maintenance updates can be viewed in three places. 
+
+1. Service overview page in `the Aiven Console <https://console.aiven.io/>`_.
```

Comment:
Please
* Rephrase to ``Service's **Overview** page in `Aiven Console <https://console.aiven.io/>`_`` (GUI items always within ``**``)
* Use the unordered list (bullets) (the order of the items in the list is not significant).
* Remove the full stops at the ends of all three list items (the list items are not full sentences).

Reply:
Also done. 

Also, I finally found the style guide. I'll make sure to review this for future docs updates. If this isn't the right style guide, lemme know. :) 
https://aiven.slab.com/posts/style-guide-resources-for-editors-dag2nrwl

Reply:
Actually the most relevant is https://github.com/aiven/devportal/blob/main/CONTRIBUTING.rst :-)